views/showcase.html

```diff
@@ -180,6 +168,20 @@ <h2 class="bg-gray-200 px-4 py-3 font-bold mt-4 mb-1">{{__('Metadata')}}</h2>
       </div>
       <!-- Metadata -->
 
+      <!-- Keywords section -->
+        <h2 class="bg-gray-200 px-4 py-3 font-bold mt-4 mb-1">{{__('Keywords')}}</h2>
+        {% if dataset.keywords and dataset.keywords.length !== 0 %}
+          <div class="container">
+            <div class="flex flex-wrap ">
+            {% for t in dataset.keywords %}
+              <a href="/search?q=tags:{{t | safe}}" class="h-5 px-1 py-0 text-primary text-xs border rounded-full border-primary mr-3 mb-3 mt-3 lowercase hover:bg-secondary">{{t | safe}}</a>
```

Comment:
This is great - the only thing is probably we could increase paddings so that the badges look better.

Reply:
I increased paddings and size of keywords badges and now it looks like 
![keywords_new](https://user-images.githubusercontent.com/12686547/71573190-e3243d80-2ae2-11ea-8953-e594cd9f1838.png)